highscore.txt

```diff
@@ -1,2 +0,0 @@
-24
-Yash
```

Comment:
Why are we removing this?

Reply:
This was unintended, I will close this PR and make 2 new PRs